frappe/patches/v12_0/set_correct_url_in_files.py

```diff
@@ -0,0 +1,44 @@
+import frappe
+import os
+
+def execute():
+	files = frappe.get_all('File', filters={'is_folder': 0})
```

Comment:
you can change the line to this:

```suggestion
	files = frappe.get_all('File', filters={'is_folder': 0}, ["name", "file_name", "file_url"])
```

and then you shouldn't need `get_doc()` inside the for loop. That should speed things up by a lot. To get full path for the file you can then use `os.abs_path()` instead of `doc.get_full_path()`